docs/buildtest_perlmutter.rst

```diff
@@ -72,37 +118,38 @@ In this exercise you will learn how to use the :ref:`buildspec_interface`. Let's
 
 In this task you will be required to do the following
 
+**TODO:**
+
 1. Find all tags
-2. List all filter and format fields
-3. Format table via fields ``name``, ``description``
-4. Filter buildspec by tag ``e4s``
+2. List all filters and format fields
+3. Format tables via fields ``name``, ``description``
+4. Filter buildspecs by tag ``e4s``
 5. List all invalid buildspecs
 6. Validate all buildspecs by tag ``e4s``
 7. Show content of test ``hello_world_openmp``
 
 Exercise 4: Querying Test Reports
 ----------------------------------
 
-In this exercise you will be learn how to :ref:`query test report <test_reports>`. This can be done by
+In this exercise you will be learn how to :ref:`query test reports <test_reports>`. This can be done by
 running ``buildtest report``. In this task please do the following
 
-1. List all filter and format fields
-2. Query all test by returncode 0
-3. Query all test by tag ``e4s``
-4. Print total count of failed tests
+1. List all filters and format fields
+2. Query all tests by returncode 0
+3. Query all tests by tag ``e4s``
+4. Print total count of all failed tests
 
-Let's upload the test to CDASH by running the following::
+Let's upload the tests to CDASH by running the following::
 
     buildtest cdash upload $USER-buildtest-tutorial
 
-Take some time to analyze the output in CDASH by opening the link including PASS/FAIL test.
+Take some time to analyze the output in CDASH by opening the link.
```

Comment:
What link? Is it provided by buildtest? If so, say that here.

Reply:
yeah the link is provided in output of `buildtest cdash upload`. They can also do `buildtest cdash upload -o` which will automatically open the link but on Perlmutter most likely they need to run this from X11 forwarding. The output shows link.

A typical example would look something like this

```
(buildtest)  ~/Documents/github/buildtest/ [fix_perlmutter_tutorial_feedbacks_justin] buildtest cdash upload demo       
Reading report file:  /Users/siddiq90/Documents/github/buildtest/var/report.json
Uploading 110 tests
Build Name:  demo
site:  generic
MD5SUM: 66aab132ac8c51e283d37bf5b247739b
You can view the results at: https://my.cdash.org//viewTest.php?buildid=2278324

```

Reply:
That's fine. Just say something like 'the link to cdash provided by buildtest build' or whatever is generating it. That way they have some context

Reply:
ok i did clarify this with hyperlink to `buildtest cdash` section. This is good point you bring up since `buildtest cdash` was not covered in this page so this would be new for users.